dongle_app/js/app.js

```diff
@@ -87,7 +88,13 @@ window.onload = function() {
     SimplePush.createChannel(
       'foxcast-notifications',
       function onNotification(version) {
+        console.log('isNotificationHandled ' + isNotificationHandled);
+        if (isNotificationHandled) {
```

Comment:
We can test to do: FoxPlayer.changeSrc() if there is a new url. What do you think?
https://github.com/borjasalguero/foxcast/blob/master/dongle_app/libs/fox_player.js#L36